### PR TITLE
feat: add navbar link on `VtmnNavbar` component

### LINKED ISSUE #1361: feat: add navbar link on VtmnNavbar component
**Describe the solution you'd like**

For details & motivation, please see https://github.com/Decathlon/vitamin-design/issues/158

### PR DESCRIPTION
**Pull request by lauthieb**
## Changes description

Closes #1361 

## Checklist

- [x] Make sure you are requesting to **pull a topic/feature/bugfix branch**. Please, don't request directly from your main!
- [x] Check commits & PR names matches our requested structure. It must follow the https://www.conventionalcommits.org pattern.
- [x] Check your code additions will fail neither code linting checks.
- [x] I have reviewed the submitted code.
- [x] I have tested on related showcases.
- [ ] If it includes design changes, please ask for a review `design-system-core-team-design` GitHub team.

## Does this introduce a breaking change?

- No

### CODE REVIEW

**Comment by thibault-mahe** [comment]
Thanks @lauthieb!
Quick question while I'm reviewing this PR, do we plan or anticipate other kind of buttons, for instance:

**Review by thibault-mahe — APPROVED** [review]
Almost completely ok for me, I just put it momentarily  on standby because I can't start the storybook of the svelte version currently :/

File: packages/sources/svelte/src/components/indicators/VtmnProgressbar/VtmnProgressbarLinear.svelte [hunk 1]
```diff
@@ -84,3 +84,17 @@
...
+    width: 1px;
+    height: 1px;
+    padding: 0;
+    margin: -1px;
+    overflow: hidden;
+    clip: rect(0, 0, 0, 0);
+    white-space: nowrap;
+    border-width: 0;
+  }
+</style>
```
[the remark this answers is not in the document]
Comment (lauthieb) [reply packages/sources/svelte/src/components/indicators/VtmnProgressbar/VtmnProgressbarLinear.svelte:100]: Hi @thibault-mahe thanks for your suggestions, I will explain tomorrow you why we did this after our a11y audit. For now, I resolve as it's ok for you.

**Comment by lauthieb** [comment]
> Thanks @lauthieb! Quick question while I'm reviewing this PR, do we plan or anticipate other kind of buttons, for instance: image (India) image (South Africa) ...

I don't think so as we need to rationalize and avoid having different UI for navbar per domain. I just sent a chat to our @Decathlon/design-system-core-team-design.

**Pull request merged**